helm/cluster/values.schema.json

```diff
@@ -39,22 +39,42 @@
                                 "title": "Name",
                                 "description": "Name of the config map or secret. The object must exist in the same namespace as the cluster App."
                             },
-                            "optional": {
-                                "type": "boolean",
-                                "title": "Optional",
-                                "description": "Optional marks this ValuesReference as optional. When set, a not found error for the values reference is ignored, but any ValuesKey, TargetPath or transient error will still result in a reconciliation failure."
+                            "priority": {
+                                "type": "integer",
+                                "title": "Priority",
+                                "default": 25,
+                                "max": 150,
```

Comment:
Why do you have max 150? Is this arbitrary?

Reply:
According to our [docs](https://docs.giantswarm.io/vintage/getting-started/app-platform/app-configuration/#extra-configs) the max priority of an extraConfig is 150.